juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule

```diff
@@ -0,0 +1,254 @@
+    topic vpn.l3vpn {
+        description "Refer routing instance OSPF session state and PE interfaces state and notify anomalies";
+        synopsis "L3VPN OSPF IPv6 session state analyzer";
+        rule check-l3vpn-ospf3-state {
+            synopsis "L3VPN session state detector";
+            description "Refer L3VPN peer state, PE interface state from rule routing.ospf/check-ospf3-neighbor-state & interfaces/check-ifl-state and notify anomalies when flap count increases";
+            rule-frequency 185s;
+            network-rule;
+            field instance-ifl-no {
+                constant {
+                    value "{{pe-ifl-number}}";
+                }
+                type integer;
+                description "Routing instance IFL number";
+            }
+            field instance-interface-name {
+                constant {
+                    value "{{pe-interface-name}}";
+                }
+                type string;
+                description "Routing instance interface name";
+            }
+            field instance-interface-status {
+                reference {
+                    path "/device-group[device-group-name={{pe-device-group}}]/device[device-id={{pe-device-name}}]/topic[topic-name='interfaces']/rule[rule-name=check-ifl-state]/field[sub-interface-index='{{pe-ifl-number}}' and interface-name='{{pe-interface-name}}']/ifl-oper-status";
+                }
+                type string;
+                description "Routing instance interface status";
+                enumerate UP as 1;
+                enumerate DOWN as 0;
+                enumerate TESTING as -1;
+                enumerate UNKNOWN as -2;
+                enumerate DORMANT as -3;
+                enumerate NOT_PRESENT as -4;
+                enumerate LOWER_LAYER_DOWN as -5;
+            }
+            field neighbor-session {
+                constant {
+                    value "{{customer-neighbor-address}}";
+                }
+                type string;
+                description "Protocol neighbor address";
+            }
+            field pe-router-name {
+                constant {
+                    value "{{pe-device-name}}";
+                }
+                type string;
+                description "PE router name to monitor";
+            }
+            field vpn-name {
+                constant {
+                    value "{{customer-vpn-name}}";
+                }
+                type string;
+                description "VRF name";
+            }
+            field vpn-state {
+                reference {
+                    path "/device-group[device-group-name={{pe-device-group}}]/device[device-id={{pe-device-name}}]/topic[topic-name='routing.ospf']/rule[rule-name=check-ospf3-neighbor-state]/field[neighbor-address='{{customer-neighbor-address}}']/ospf-neighbor-state";
+                    data-if-missing {
+                        value DOWN;
+                    }
+                    time-range 185s;
```

Comment:
Can we use little bigger time range? 1.5 times?

Reply:
made it 278s